packages/react/async-boundary/src/withAsyncBoundary.tsx

```diff
@@ -17,7 +17,7 @@ import AsyncBoundary from './AsyncBoundary';
  * @see https://jbee.io/react/error-declarative-handling-1/ 선언적으로 로딩과 에러 상태 처리하기
  * @see https://toss.im/slash-21/sessions/3-1 Suspense와 에러 처리 관련된 Slash 21 발표
  */
-export default function withAsyncBoundary<Props = {}>(
+export default function withAsyncBoundary<Props extends Record<string, unknown> = Record<string, never>>(
```

Comment:
I think any constraints are not required here because all React developers might know props should be object, and even if other types were passed it wouldn't work. I think that's why React doesn't provide any constraint in `ComponentType` too.

But, apart from that, this change looks good to me.